doc/data-structures.rst

```diff
@@ -549,6 +549,14 @@ binary operations that act on xarray objects. In the future, we hope to write
 more helper functions so that you can easily make your functions act like
 xarray's built-in arithmetic.
 
+CF-compliant coordinate variables
```

Comment:
I'm definitely a fan of calling out other packages that add useful functionality on top of xarray, but I'm not sure we want this in the doc page on generic data structures. Maybe a separate page for "Weather and Climate Add-ons"?